src/huggingface_hub/keras_mixin.py

```diff
@@ -174,6 +179,7 @@ def push_to_hub_keras(
         **model_save_kwargs,
     )
 
+    copytree(tensorboard_dir, f"{repo_path_or_name}/logs")
```

Comment:
A lot of tests are failing because `tensorboard_dir` is None by default. You should only do this step when it's not `None`

See https://github.com/huggingface/huggingface_hub/runs/5067079206?check_suite_focus=true#step:5:209